xmlStyle.go

```diff
@@ -80,18 +80,18 @@ type xlsxStyleSheet struct {
 	CellXfs      xlsxCellXfs       `xml:"cellXfs,omitempty"`
 	NumFmts      xlsxNumFmts       `xml:"numFmts,omitempty"`
 
-	theme          *theme
+	theme *theme
+
+	sync.RWMutex   // protects the following
 	styleCache     map[int]*Style
 	numFmtRefTable map[int]xlsxNumFmt
-	lock           *sync.RWMutex
```

Comment:
The lock is still there, just moved above what it protects. (Only places it was used internally was for the maps.)